src/polygon.c

```diff
@@ -906,6 +907,51 @@ spherepoly_in(PG_FUNCTION_ARGS)
 	PG_RETURN_POINTER(poly);
 }
 
+Datum
+spherepoly_rad(PG_FUNCTION_ARGS)
```

Comment:
I think, in this implementation we do not properly process arrays with some NULL elements. There is a number of functions like ARR_HASNULL, ARR_NULLBITMAP that may help to identify NULL array elements. The simple fix is to reject such arrays with NULL elements.

It seems for spherepoly_deg we missed the check for NULL array elements as well.

I'm ok to mark it as PARALLEL SAFE and STRICT. I think we mssed to mark spherepoly_deg because it was under development when we were adding PARALLEL SAFE and STRICT.

Reply:
@vitcpp : I think I have addressed all of your review items. Please check. I have left the commits unsquashed to facilitate your review. I will rebase and squash the commits before merging.